scitos_mira/src/ScitosDrive.cpp

```diff
@@ -65,7 +66,7 @@ void ScitosDrive::initialize() {
 }
 
 void ScitosDrive::velocity_command_callback(const geometry_msgs::Twist::ConstPtr& msg) {
-  if (! barrier_status_.barrier_stopped ) {
+  if (! barrier_status_.barrier_stopped || ! emergency_stop_.data) {
```

Comment:
I think it is the other way round. Before this change, `/cmd_vel` velocity commands are sent on to the MIRA side if barrier_stopped is NOT true, otherwise they are ignored. This change will make them be sent on to MIRA if either the barrier stop is false or the emergency stop is false, so the barrier stop could be on but the emergency stop not so the commands are sent. The if is not saying "if ... then stop", it says "if ... then go ahead".